packages/patternfly-4/react-core/src/components/Accordion/AccordionContent.tsx

```diff
@@ -1,8 +1,9 @@
 import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Accordion/accordion';
+import {AccordionContext} from './AccordionContext';
 
-export interface AccordionContentProps extends React.HTMLProps<HTMLElement> {
+export interface AccordionContentProps extends React.HTMLProps<HTMLDivElement> {
```

Comment:
Looks like this could be HTMLDivElement or the equivalent for `dd`. Not sure how this could be typed/handled. Any ideas? @redallen @seanforyou23